Header header
int32 leg_num
bool foot_off_gnd
geometry_msgs/Point foot_pos
sensor_msgs/JointState joint_ang
